src/js/models/card-node.js

```diff
@@ -35,7 +35,8 @@ export default class CardNode {
         this.display();
       },
       cancel: () => this.display(),
-      remove: () => this.remove()
+      remove: () => this.remove(),
+      card: this
```

Comment:
I see how this is being used.

Could this expose the abstract node instead of the `CardNode`? The abstracts are already public or at the least "intimate" API, and unless I've read incorrectly the only other thing needed in https://github.com/bustlelabs/typeset.bustle.com/pull/21/files#diff-04c1550a2081f24687e1031a3bae973fR13 is the `editor` itself, which could be passed directly into the component.